TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md

```diff
@@ -0,0 +1,73 @@
+TWG Meeting Report
+Nov 30/2020
+
+## TG Reports
+
+### SW TG
+
+Written report prepared - thanks to Jeremy:
+https://github.com/openhwgroup/core-v-docs/blob/master/program/TG-reports-for-TWG/20201130-sw-tg.md
+
+IDE - Nearing CQ. Most important issue is "hello world application".
+
+GNU Tools - Good progress. Need to have FSF assignment from contributors - Action to Duncan. 
+FSF assignment from ETH Z needs to be requested. Rick to help out. Target board needed. 
+
+CLANG/LLVM - PPL report below
+
+FreeRTOS - PPL report below
+
+
+### HW TG
+Verbal report:
+
+CORE-V-MCU – Quicklogic is putting resources in place. Need to review against original PPL plan and schedule PL.
+
+CORE-V-VEC – Passed PPL (Nov 23). Research proposal submitted to Mitacs and feedback coming in January.
+
+CORE-V-FPGA - missed the summary - please provide, Hugh

```

Comment:
CORE-V-MCU FPGA - waiting for RTL freeze for CV32E40P core to update platform. Also anticipating using FreeRTOS as the SW development environment for CORE-V-MCU FPGA and SOC.